src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Figure implements FigureInterface {

```

Comment:
Do we need to be able to create instances of Figure class?

Reply:
ok. understand. it should be as protected

Reply:
No, it should be abstract